pkg/console/operator/sync_v400.go

```diff
@@ -86,10 +86,10 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	if customLogoError != nil {
 		msg := fmt.Sprintf("%q: %v", "customlogoconfigmap", customLogoError)
 		klog.V(4).Infof("incomplete sync: %v", msg)
```

Comment:
eventually the logging could be part of the `handleCondition `

Reply:
Thats not a bad idea.  I dislike all the noise everywhere. We should factor the logging noise out.